lighthouse-plugin-ad-speed-insights/audits/ad-request-critical-path.js

```diff
@@ -176,11 +218,13 @@ class AdRequestCriticalPath extends Audit {
     }
     const pageStartTime = getPageStartTime(networkRecords);
     let tableView = blockingRequests.map((req) => ({
-      url: requestName(req.url),
+      url: trimQuery(req.url),
+      abbreviatedUrl: getAbbreviatedUrl(req.url),
+      type: req.resourceType,
       startTime: (req.startTime - pageStartTime) * 1000,
       endTime: (req.endTime - pageStartTime) * 1000,
       duration: (req.endTime - req.startTime) * 1000,
-    }));
+    })).filter((r) => r.duration > 30 && r.startTime > 0);
```

Comment:
Your current PR description says 20 ms, can you update it?

Reply:
Done